docs/index.asciidoc

```diff
@@ -204,12 +204,16 @@ Whether results should be sorted or not
 An array of fields to copy from the old event (found via elasticsearch) into the
 new event, currently being processed.
 
-In the following example, the value of `@timestamp` on the event found via
-elasticsearch is copied to the current event's `started` field:
+In the following example, the values of `@timestamp` and `event_id` on the event
+found via elasticsearch are copied to the current event's
+`started` and `start_id` fields, respectively:
 
 [source,ruby]
 --------------------------------------------------
-fields => { "@timestamp" => "started" }
+fields => {
+  "@timestamp" => "started"
+  "event_id" => "start_id"
+}
 --------------------------------------------------
```

Comment:
@robbavey How about this?

Reply:
@webmat Yes, I think that is fine

Reply:
The only weird thing is that the documentation states `array` and that is a `hash`, but I guess that is down to the old array of tuples syntax?

Reply:
Yes. It's a question I'll bring up at the next sync.

I'm wondering what the process is to move from the deprecated array to hash, for arrays that work like this attribute (a series of tuples). Unfortunately just making this a hash doesn't work. Hash attributes don't support arrays of tuples, so it looks like this requires a breaking change...

Reply:
At least arrays support the hash notation... Although this causes never-ending confusion :-)